server/lib/nl/detection/date.py

```diff
@@ -86,27 +87,27 @@ def parse_date(query: str, ctr: Counters) -> DateClassificationAttributes:
       count, = match.groups()
       count_num = _YEARS_STRING_TO_NUM.get(count, count)
       year = datetime.date.today().year
-      dates.append(Date('before', year - 1, year_span=int(count_num)))
+      dates.append(Date(_LAST_YEARS_PREP, year, year_span=int(count_num)))
       trigger_strings.append(query[match.start():match.end()])
 
   for pattern in LAST_YEAR:
```

Comment:
Maybe add some comments about what is going on here?

Reply:
done